ci/authn-k8s/dev/dev_conjur_log_volume.template.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: v1
+kind: PersistentVolume
```

Comment:
Looking at the GKE docs, you probably don't need to create the PV explicitly. Create the PVC and the PV will be provisoned by GKE: https://cloud.google.com/kubernetes-engine/docs/concepts/persistent-volumes#dynamic_provisioning

Reply:
@hughsaunders do we want to make this solution generic? So it would work the same in our OpenShift environments as well?

Reply:
I agree with Rafi here. It seems that dynamic provisioning will work only for GKE.

Reply:
This is a bug with our openshift deployment, but I do agree that we should use a method that works for both.

Reply:
i get a page not found for the link above. So @hughsaunders do you still think we should use dynamic provisioning?